scripts/builddown

```diff
@@ -11,4 +11,4 @@ export NODE_OPTIONS="--max-old-space-size=4096 ${NODE_OPTIONS:-}"
 scriptdir=$(cd $(dirname $0) && pwd)
 export PATH=$(cd $scriptdir && npm bin):$PATH
 scope=$(${scriptdir}/current-scope)
-exec lerna run build --scope ${scope} --include-dependents
+exec lerna run ${1:-build} --scope ${scope} --include-dependents
```

Comment:
Wouldn't you use `foreach` instead?

Reply:
Well that builds everything and this just builds downstream packages.